theSrc/scripts/LabeledScatter.js

```diff
@@ -582,7 +582,7 @@ class LabeledScatter {
         .select('.cursor-pointer')
         .attr('x', 0)
         .attr('y', 0)
-        .attr('transform', `translate(${0.5 * this.width},${this.height - footer_height - footer_padding})`)
+        .attr('transform', `translate(${this.titleX('Center of plot area')},${this.height - footer_height - footer_padding})`)
```

Comment:
Why is this always using `this.titleX('Center of plot area')`? What if I change the alignment of the footer?

Reply:
The footer doesn't have an alignment setting. I could include it but it won't be exposed to users at the moment. We could include it in the future but I thought eventually footers will be drawn by Displayr.

Reply:
Ah right. I forgot that the footers will eventually be drawn by Displayr